OpenAISwift.podspec

```diff
@@ -0,0 +1,21 @@
+Pod::Spec.new do |s|
+  s.name = 'OpenAISwift'
+  s.version = '0.0.1'
+  s.summary = 'OpenAISwift'
+  s.description  = <<-DESC
+  commit:${commit}
+  DESC
+  s.license = 'MIT'
+  s.homepage = 'https://github.com/KittenYang/OpenAISwift'
```

Comment:
I suspect that you leave this by error, you should point to the original repo not the forked one